components/FeatureHighlight/src/MDCFeatureHighlightViewController.m

```diff
@@ -232,6 +233,20 @@ - (void)dismiss:(BOOL)accepted {
                            }];
 }
 
+- (void)applyMDCFeatureHighlightViewDefaults {
+  _outerHighlightColor = [self MDCFeatureHighlightDefaultOuterHighlightColor];
+  _innerHighlightColor = [self MDCFeatureHighlightDefaultInnerHighlightColor];
+}
```

Comment:
Similar to the viewDidLoad logic above, please consider moving this into the init method.